plugins/inputs/t128_lte/README.md

```diff
@@ -0,0 +1,11 @@
+# 128T LTE Input Plugin
+
+The lte input plugin collects data from a 128T.
```

Comment:
This is a public facing doc so we need to boost this up a bit. We can simply lift up the blurb from [here](https://docs.128technology.com/docs/plugin_monitoring_agent#lte-collector)